boards/x86/acrn/doc/index.rst

```diff
@@ -0,0 +1,30 @@
+.. _acrn:
+
+ACRN UOS (Guest Operating System)
+#################################
+
+Overview
+********
+
+This baseline configuration is intended to be used as a starting point for
+custom ACRN UOS configurations. It supports the following devices:
+
+* I/O APIC
+* local APIC timer
+* NS16550 UARTs
+
+Serial Ports
+------------
+
+The serial ports are assumed present at traditional COM1: and COM2: I/O-space
+addresses (based at 0x3f8 and 0x2f8, respectively). Only polled operation is
+supported in this baseline configuration, as IRQ assignments under ACRN are
+configurable (and frequently non-standard). Interrupt-driven and MMIO operation
+are both theoretically possible, however.
+
+Running
+*******
+
+The output from the build process is a zephyr.elf file, which is loaded
```

Comment:
Add an example of how to build one of the existing sample applications (e.g., hello world), or do you have some other UOS sample that's more ACRN specific you can supply?